core-java-modules/core-java-21/src/test/java/com/baeldung/java21/SwitchPatternTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.java21;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class SwitchPatternTest {
+
+    @Test
+    void givenDifferentSqitchPattern_whenCalledBoth_thenReturnTheSameResult() {
```

Comment:
There's a typo Switch instead of Sqitch
```suggestion
    void givenDifferentSwitchPattern_whenCalledBoth_thenReturnTheSameResult() {
```